osmtm/utils.py

```diff
@@ -109,3 +111,19 @@ def convert_to_multipolygon(geoms):
     geom2d = shapely.wkt.loads(wkt2d)
 
     return geom2d
+
+
+def interpolate_text(text, properties):
```

Comment:
This is simply re-implementing Python's existing `str.format()` function. We should just use that.

```python
>>> '{foo} is bar'.format(foo='bar')
'bar is bar'
```